src/main/java/org/spongepowered/api/command/args/GenericArguments.java

```diff
@@ -921,22 +922,35 @@ public static CommandElement bool(Text key) {
 
     private static class EnumValueElement<T extends Enum<T>> extends PatternMatchingCommandElement {
         private final Class<T> type;
+        private final Map<String, T> values;
```

Comment:
Why should it be immutable? The code never modifies it in this pull, and the map is already private.